.github/workflows/restart-search-service.yml

```diff
@@ -0,0 +1,26 @@
+name: SSH to VMs and Restart search service
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Connect to VM and execute commands
```

Comment:
this will be difficult to scale to the many current deployments. it also won't work for deployments behind a VPN. Sorry, but I see no other options than to use specific internal Jenkins jobs.

Reply:
Indeed, I forgot about the VPN issue. Do you already know how to handle VPN with Jenkins? I'm closing this draft.